package.json

```diff
@@ -111,6 +109,7 @@
     "@testing-library/react": "^11.2.3",
     "@testing-library/react-hooks": "^3.7.0",
     "@testing-library/user-event": "^12.6.0",
+    "@web-stories-wp/date": "*",
     "@web-stories-wp/fonts": "*",
     "@web-stories-wp/jest-amp": "*",
     "@web-stories-wp/jest-puppeteer-amp": "*",
```

Comment:
Let's move those to `dependencies` instead of `devDependencies`

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6193/commits/eb0667f8d6d1f10ebb0d0fc70c14391041aa3dbe